mlrun/config.py

```diff
@@ -242,6 +242,14 @@
             "channel": "master",
         },
     },
+    "storage": {
+        # What type of auto-mount to use for functions. Can be one of: none, auto, v3io_credentials, v3io_fuse, pvc.
+        # Default is auto - which is v3io_credentials when running on Iguazio and pvc otherwise (MLRun kit)
```

Comment:
I feel like it should be else pvc if X is configured (need to think what is that X) else None